langserve/server.py

```diff
@@ -487,6 +505,8 @@ async def _get_config_and_input(
                 body.config,
                 keys=config_keys,
                 model=ConfigPayload,
+                request=request,
```

Comment:
nit: python convention is to only invoke named arguments by name and positional arguments by position